### ISSUE TITLE
Prompt user on which tag to resume

### ISSUE THREAD

**Issue by Blackbeing** [ayoisaiah/focus#11]
So I started using focus for  tracking different work I do.  For tracking one work, it was good, just `focus resume` and go right back.
Now with multiple tags, mostly I just forget and do `focus resume`, only later on to realize I've been tracking the wrong work and have to edit the tags manually.

Suggestion: On focus resume, prompt user on which tag to resume on already available tags

The only problem that I noticed currently is that when you change the tag during resume, it continues where the previous tag stopped, even if it's a different tag. Maybe we can keep track of each tag "stopped time" and if you resume a specific tag, it continues from where the tag was stopped.

**Comment by ayoisaiah** [comment]
@Blackbeing Good suggestions! I will work on implementing them this week

**Comment by ayoisaiah** [comment]
I've implemented your feature request. `focus resume` will always resume the last paused session (as before), but a new `--select/-s` flag has been added to resume a different session. 

In a related change, the tags are now displayed in the countdown.

![image](https://user-images.githubusercontent.com/13372163/219578363-1ac9917b-1f63-4611-9371-2887613d4741.png)

https://github.com/ayoisaiah/focus/commit/b6b5183b6c79e4b740dbb4f020540a13668c18d8
